src/NuGet.Core/NuGet.Common/PathUtil/PathResolver.cs

```diff
@@ -173,17 +176,18 @@ internal static string GetPathToEnumerateFrom(string basePath, string searchPath
 
         internal static string NormalizeBasePath(string basePath, ref string searchPath)
         {
-            const string relativePath = @"..\";
+            string parentDirectoryPath = $"..{Path.DirectorySeparatorChar}";
+            string currentDirectoryPath = $".{Path.DirectorySeparatorChar}";
 
             // If no base path is provided, use the current directory.
-            basePath = String.IsNullOrEmpty(basePath) ? @".\" : basePath;
+            basePath = String.IsNullOrEmpty(basePath) ? currentDirectoryPath : basePath;
 
-            // If the search path is relative, transfer the ..\ portion to the base path. 
+            // If the search path is relative, transfer the parent directory portion to the base path.
             // This needs to be done because the base path determines the root for our enumeration.
-            while (searchPath.StartsWith(relativePath, StringComparison.OrdinalIgnoreCase))
+            while (searchPath.StartsWith(parentDirectoryPath, StringComparison.OrdinalIgnoreCase))
```

Comment:
In this specific case I think it's benign, since the "search for" strings `../` and `..\` are unaffected by casing.